src/components/Socials.tsx

```diff
@@ -0,0 +1,52 @@
+import styled from "styled-components";
+
+import facebook from '../assets/icons/facebook.svg';
+import instagram from '../assets/icons/instagram.svg';
+import youtube from '../assets/icons/youtube.svg';
+
+const SocialsWrapper = styled.div`
+  display: flex;
+  flex-direction: row;
+  margin-right: 60px;
+  align-items: center;
+`
+
+const SocialImage = styled.img`
+  margin-right: 30px;
+`
+
+enum SocialType {
+    instagram,
+    facebook,
+    youtube
+}
+
+interface ISocialItem {
+    type: SocialType;
+    link: string;
+    alt: string;
+}
+
+const SocialItems: Array<ISocialItem> = [
```

Comment:
here you can also use a shorter way (optional):
`const SocialItems: ISocialItem[] = [...]`